frontend/src/metabase/reference/containers/FieldXray.jsx

```diff
@@ -0,0 +1,156 @@
+/* @flow */
+import React, { Component } from 'react'
+import Visualization from 'metabase/visualizations/components/Visualization'
+
+import { connect } from 'react-redux'
+
+import type { Field } from 'metabase/meta/types/Field'
+
+type Props = {
+    fetchXrayData: () => void,
+    field: Field,
+    xray: {}
+}
+
+class FieldXray extends Component {
+    props: Props
+
+    componentWillMount() {
+        this.props.fetchXrayData(this.props.field.id)
+    }
+    render () {
+        return (
+            <div style={{
+                maxWidth: 550,
+                marginLeft: 'calc(48px + 1rem)',
+            }}>
+                <h3>XRAY</h3>
+                <div className="Grid Grid--1of2 Grid--gutters mt1">
+                    <div className="Grid-cell">
+                        <div className="bordered rounded shadowed py2">
+                            <Visualization
+                                series={[this.props.xray.min]}
+                            />
+                        </div>
+                    </div>
+                    <div className="Grid-cell">
+                        <div className="bordered rounded shadowed py2">
+                            <Visualization
+                                series={[this.props.xray.max]}
+                            />
+                        </div>
+                    </div>
+                </div>
+                <Visualization
```

Comment:
@salsakran I figured it'd be easiest to just re-use our visualization code wherever we can so we don't end up with different visualization code paths or bugs specific to the xrays. Is there something specific about the histogram x-ray that using lower level dc would help with?